analyze/algorithm/DiffObjectDetector.cpp

```diff
@@ -36,14 +37,16 @@ namespace am {
 					}
 				}
 				if (nextCheck.size())
-					bfs(changes, nextCheck, object, col);
+					bfs(changes, nextCheck, object, col, minCount);
 
 				return object;
 			}
 
-			std::vector<Pixels> startObjectsSearch(MatrixU16& changes, size_t step, const Column& col)
+			std::vector<Pixels> startObjectsSearch(MatrixU16& changes, am::configuration::Configuration* conf, const Column& col)

```

Comment:
pass by raw pointer, we are not going to destroy Configuration object in this function or in another thread.
The comparison ends only then all Conf, Logger can be destroyed normally